source/GameInterface/Services/Modules/Validators/ModuleValidator.cs

```diff
@@ -0,0 +1,34 @@
+using System;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace GameInterface.Services.Modules.Validators;
+
+public class ModuleValidator : IModuleValidator
```

Comment:
I found it a lot easier to have the interface and implementation in the same file,

I suggest moving the IModuleValidator interface definition to this file. Also could you add
```CS
/// <inheritdoc href="IModuleValidator">
```
And also add a summary comment for the IModuleValidator interface

Reply:
I adapted it to the code conventions of the project